EssentialCSharp.Web.Tests/Integration/CaptchaTests.cs

```diff
@@ -23,8 +26,9 @@ public async Task CaptchaService_Verify_Success()
         string hCaptchaSecret = "0x0000000000000000000000000000000000000000";
         string hCaptchaToken = "10000000-aaaa-bbbb-cccc-000000000001";
         string hCaptchaSiteKey = "10000000-ffff-ffff-ffff-000000000001";
-        HttpResponseMessage response = await captchaService.Verify(hCaptchaSecret, hCaptchaToken, hCaptchaSiteKey);
+        HCaptchaResult? response = await captchaService.Verify(hCaptchaSecret, hCaptchaToken, hCaptchaSiteKey);
 
-        Assert.Equal(HttpStatusCode.OK, response.StatusCode);
+        Assert.NotNull(response);

```

Comment:
```suggestion
        Assert.True(response?.Success);

```

Reply:
Doesn't this do the same thing?

Reply:
Yes, functionally the same thing in terms of the test passing or not. I personally like splitting it up because the error message is better, and I can quickly identify if the response is null or if it isn't successful, and in doing just one assert, I believe the message is the same as if the condition passed into True is false.